rest/replicatortest/replicator_test.go

```diff
@@ -2503,12 +2503,12 @@ func TestActiveReplicatorPullFromCheckpoint(t *testing.T) {
 	assert.Equal(t, int64(numRT2DocsInitial), pullCheckpointer.Stats().ExpectedSequenceCount)
 
 	// checkpoint assertions
-
-	// CBG-2767 skip assertions for GetCheckpoint stats
+	// CBG-2788 skip assertions till this ticket is implemented
 	if rt2.GetDatabase().OnlyDefaultCollection() {
 		assert.Equal(t, int64(0), pullCheckpointer.Stats().GetCheckpointHitCount)
 		assert.Equal(t, int64(1), pullCheckpointer.Stats().GetCheckpointMissCount)
 	}
```

Comment:
I was expecting these stats to work even without the work to fetch local checkpoints as well as remote, since fetching remote still should work the same, as long as there are no rollbacks/bucket flushes.

Is that not true? Do we know why?

Reply:
Ah I'm stupid. Got soem of the logic wrong I think and for some reason in my head put it down to rollback cases etc. Apologies I'm updating that now.